src/features/query-bar/components/filter/FilterButton.js

```diff
@@ -239,7 +239,7 @@ class FilterButton extends React.Component<Props, State> {
 
         const buttonClasses = classNames('query-bar-button', numberOfAppliedConditions !== 0 ? 'is-active' : '');
 
-        const isFilterDisabled = columns.length === 0;
+        const isFilterDisabled = columns === undefined || columns.length === 0;
```

Comment:
Should we cover columns === null?

If so, you can replace "columns === undefined" with "!columns".

Reply:
Yes, it's possible that a developer could forget to pass in columns